api/src/services/__tests__/meetingsService.ts

```diff
@@ -110,6 +111,19 @@ describe('meetingsService', () => {
     });
   });
 
+  describe('participantExists', () => {
+    it('should query for the participant with the given id and associated meeting with given id', async () => {
+      const meetingId = 2;
+      const principalId = 3;
+      mockQuery(
+        'select `principal_id` from `participants` where `meeting_id` = ? and `principal_id` = ?',
+        [meetingId, principalId],
+        [3]
+      );
+      expect(await participantExists(meetingId, principalId)).toEqual(true);
```

Comment:
The test only checks for the happy path. Should there also be a test for the sad path?

Reply:
ok will add the test now